jf_agent/data_manifests/git/adapters/github.py

```diff
@@ -21,15 +21,32 @@ class GithubManifestGenerator(ManifestAdapter):
     '''
 
     def __init__(
-        self, token: str, company: str, org: str, instance: str, verify=True, **kwargs
+        self,
+        token: str,
+        base_url: str,
+        company: str,
+        org: str,
+        instance: str,
+        verify=True,
+        **kwargs,
     ) -> None:
         # Super class fields
         self.company = company
         self.org = org
         self.instance = instance
         # Session fields
         self.token = token
-        self.base_url = 'https://api.github.com/graphql'
+
+        if not base_url:
```

Comment:
@jruel4 I'm not actually sure if base_url can ever be None or an empty string, but this is just in case